cloud/org.go

```diff
@@ -177,6 +178,9 @@ func (c *Client) CreateOrg(ctx context.Context, org string) error {
 
 // GetOrgID retrieves the org ID for a named org.
 func (c *Client) GetOrgID(ctx context.Context, orgName string) (string, error) {
```

Comment:
If all of the methods now take org names, can we unexport this method?

Reply:
unfortunately the satellite-proxy depends on an `satellite_org` metadata field being set to the ID; we need to set https://github.com/earthly/earthly/blob/acb/list-satellites-by-org-name/cmd/earthly/satellite_cmds.go#L659 which ends up being used in https://github.com/earthly/earthly/blob/acb/list-satellites-by-org-name/buildkitd/buildkitd.go#L950

Reply:
as a result we must query the org ID when a `--org ...` is passed via the cli: https://github.com/earthly/earthly/blob/acb/list-satellites-by-org-name/cmd/earthly/satellite_cmds.go#L382